src/manage_tasks.js

```diff
@@ -0,0 +1,51 @@
+class ManageTasks {
+  constructor() {
+    this.taskArr = this.loadTasksFromLocalStorage();
+  }
+
+  addTask(taskDescrption) {
+    const newTask = {
+      description: taskDescrption,
+      completed: false,
+      index: this.taskArr.length,
```

Comment:
- [ ] Please make sure that the indices of the elements start from 1 and not from 0. Currently, in the example screenshot below, your first task has an index of 0 and your third task has an index of 2. Please make sure the first task has an index of 1, the next 2 and so on.
![image](https://user-images.githubusercontent.com/45460192/229102895-cb5ccedb-9b75-4525-b9c8-3aae13236db6.png)